src/services/auth/sponsor/sponsor-router.ts

```diff
@@ -0,0 +1,118 @@
+import { Router } from "express";
+import { Database } from "../../../database";
+import RoleChecker from "../../../middleware/role-checker";
+import { Role } from "../auth-models";
+import { StatusCodes } from "http-status-codes";
+import { sendEmail } from "../../ses/ses-utils";
+import jsonwebtoken from "jsonwebtoken";
+import { Config } from "../../../config";
+import * as bcrypt from "bcrypt";
+const sponsorRouter = Router();
+
+// Get favorite events for an attendee
+sponsorRouter.get(
+    "/",
+    RoleChecker([Role.Enum.CORPORATE]),
+    async (req, res, next) => {
+        try {
+            const resumeUsers = await Database.REGISTRATION.find(
+                { hasResume: true },
+                { userId: 1 }
+            );
+            if (!resumeUsers) {
+                return res
+                    .status(StatusCodes.NOT_FOUND)
+                    .json({ error: "UserNotFound" });
+            }
+            return res.status(StatusCodes.OK).json(resumeUsers);
+        } catch (error) {
+            next(error);
+        }
+    }
+);
+
+function createSixDigitCode() {
+    let result = "";
+    const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+    for (let i = 0; i < 6; i++) {
+        result += chars.charAt(Math.floor(Math.random() * chars.length));
+    }
+    return result;
+}
+
+function encryptSixDigitCode(sixDigitCode: string): string {
+    console.log("SixDigit: ", sixDigitCode);
+    const saltRounds = 10;
+
+    try {
+        const hash = bcrypt.hashSync(sixDigitCode, saltRounds);
+        return hash;
+    } catch (err) {
+        console.error("Error encrypting the code:", err);
+        throw err;
+    }
+}
+
+sponsorRouter.post("/login", async (req, res, next) => {
+    const { email } = req.body;
```

Comment:
put thru validator. 

create 2 validators:
AuthSponsorLoginValidator
AuthSponsorVerifyValidator, and pass thru those to get the vals